front/app/containers/LandingPage/Footer.tsx

```diff
@@ -74,19 +70,20 @@ const Footer = () => {
     // tranlate header slogan into a h2 wih a fallback
     const headerSloganMultiLoc =
       appConfiguration.data.attributes.settings.core.header_slogan;
-    const genericSlogan = (
-      <FormattedMessage tagName="h2" {...messages.subtitleCity} />
-    );
     const displayHeaderAvatars =
       appConfiguration.data.attributes.settings.core.display_header_avatars;
 
     return (
       <>
         {!authUser && (
           <FooterBanner>
-            {headerSloganMultiLoc
-              ? localize(headerSloganMultiLoc)
-              : genericSlogan}
+            <FooterBannerHeading>
```

Comment:
Changed to a span as recommended by one of the a11y reports. We used the h2 here for styling more than defining the structure of our page.